nilearn/plotting/img_plotting.py

```diff
@@ -1479,9 +1479,9 @@ def plot_carpet(img, mask_img=None, detrend=True, output_file=None,
         Limit plotted voxels to those inside the provided mask (default is
         None). If not specified a new mask will be derived from data.
         See http://nilearn.github.io/manipulating_images/input_output.html.
-    detrend : :obj:`bool`, optional
```

Comment:
why is this file modified?

Reply:
Looking at the file history it seems I forgot spaces I added after parameter names to comply with the NumPy docstring formatting. I'll revert it to 923357c9bb591cd9275606cbbb25b61710554b07.